docs/current_docs/api/remote-modules.mdx

```diff
@@ -66,6 +66,96 @@ If this works, Dagger will be able to use the same credentials to access your pr
 - Azure DevOps: Use [Git Credential Manager](https://learn.microsoft.com/en-us/azure/devops/repos/git/set-up-credential-managers)
 - BitBucket: Configure credentials using the [Git credential system](https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage) (the widely-adopted implementation is [Git Credential Manager](https://github.com/git-ecosystem/git-credential-manager))
 
+### Using credentials in CI environments
+
+When running Dagger inside CI systems, you often have tokens provided by the CI provider. Instead of writing a custom helper script, you can directly store credentials in an ephemeral credentials file and configure Git to use it. Below are examples for GitLab, GitHub, and BitBucket, each shown in separate tabs. These approaches are minimal and avoid persistent storage of credentials.
+
+:::tabs
+
+:::tab[GitLab]
+
+For GitLab's `CI_JOB_TOKEN`, use `gitlab-ci-token` as the username.
+
+```yaml
```

Comment:
In our integrations pages for each CI https://deploy-preview-9147--devel-docs-dagger-io.netlify.app/integrations/ci,  we have a sub-section with an example for "SSH authentication". I propose also adding an "HTTPS authentication" (or other-named) subsection to the integration pages for GitHub Actinons, GitLab etc and also adding these examples there. To avoid duplication, these can be saved as separate code snippets and referenced. See https://github.com/dagger/dagger/blob/main/docs/current_docs/integrations/github-actions.mdx?plain=1#L29 for an example